tests/Phinx/Config/ConfigDefaultEnvironmentTest.php

```diff
@@ -24,9 +24,9 @@ public function testConfigReplacesTokensWithEnvVariables()
     {
         $_SERVER['PHINX_DBHOST'] = 'localhost';
         $_SERVER['PHINX_DBNAME'] = 'productionapp';
-        $_SERVER['PHINX_DBUSER'] = 'root';
-        $_SERVER['PHINX_DBPASS'] = 'ds6xhj1';
         $_SERVER['PHINX_DBPORT'] = '1234';
+        $_SERVER['DB_PASS'] = 'ds6xhj1';
+        $_SERVER['DB_USER'] = 'root';
```

Comment:
I think a cleared test would be to duplicate this test and make all parameters use env()